### ISSUE TITLE
Status buffer clicks don't work as expected in vi-normal-mode

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#2343]
From https://github.com/atlas-engineer/nyxt/pull/2321#issuecomment-1141246617:

n commit 1d9bd649a5d59af26227546c23265da130a47698 I tried to fix a long-standing issue with VI-mode, namely that clicking on the status bar while in vi-normal-mode does not work.

In this commit, I forward the even to the focused buffer instead of the current-buffer.

Sadly, the status buffer is focused only after the even has been sent to the renderer. So we need to click twice for it to work.

The proper fix here is to query GTK for the widget the mouse event is hovering on. I know that GTK can do this, but I can't find the API call.

Any idea?

**Comment by aartaka** [comment]
@Ambrevar, did #2321 fix it?

**Comment by Ambrevar** [comment]
Nope, this issue is unrelated and has been around forever.  We need to interact with GTK to  fix this properly.

**Comment by aadcg** [comment]
The bug may still be around, but the its formulation is most likely stale.
